lib/smarty.inc.php

```diff
@@ -16,8 +16,19 @@ function get_attribute($params) {
     $ldap_bindpw = $params["ldap_bindpw"];
     $ldap_filter = $params["ldap_filter"];
 
+    $ldapInstance = new \Ltb\Ldap(
+                                 $ldap_url,
+                                 $ldap_starttls,
+                                 isset($ldap_binddn) ? $ldap_binddn : null,
+                                 isset($ldap_bindpw) ? $ldap_bindpw : null,
+                                 isset($ldap_network_timeout) ? $ldap_network_timeout : null,
+                                 $dn,
+                                 null,
+                                 isset($ldap_krb5ccname) ? $ldap_krb5ccname : null
+                             );
+
     # Connect to LDAP
-    $ldap_connection = \Ltb\Ldap::connect($ldap_url, $ldap_starttls, $ldap_binddn, $ldap_bindpw, $ldap_network_timeout);
+    $ldap_connection = $ldapInstance->connect();
 
     $ldap = $ldap_connection[0];
```

Comment:
Maybe should we also migrate the code from this line to Ltb\LDAP, into a get_attribute method.

I can deal with this if you you are ok?

Reply:
Yes, we can do that in another PR, because this will need a new version of ltb-ldap